src/tls.cpp

```diff
@@ -26,140 +27,11 @@
  * OF SUCH DAMAGE.
  */
 
-#include "common.h"
+#include "common.h" // FIXME Circlular include/dependency?
 #include "tls.h"
 
 #if defined(DEFINED_TLS)
-
-static unsigned char cert_der[] = {
```

Comment:
Are we sure we want a generated cipher instead of a predefined one? Can it cause different performance results with different generations?

Reply:
Yes i understand that. What i mean is whether it may cause unstable performance results because between one execution of sockperf to the other execution it used a different actual key.